### DISCUSSION TITLE
`foreach` on Windows prebuilt binary

### DISCUSSION THREAD

**Discussion by rzmk** [dathere/qsv#2158] [Q&A] [answered]
I don't think there's `qsv foreach` on the Windows prebuilt binary, can it be added?

**Comment by jqnatividad** [comment, marked as answer]
Thanks for catching this @rzmk !

Added in https://github.com/jqnatividad/qsv/commit/def9c8fa98cd214f0db839b64bcd12764dcfba43

**Reply by rzmk** [reply]
~~It seems it's not enabled on the portable Windows binary too @jqnatividad:~~

https://github.com/jqnatividad/qsv/blob/e905ec91d177ad6f793625f9387e4efd2e3abd24/.github/workflows/publish-portable.yml#L70

Thanks for adding it.
